packages/mermaid-layout-elk/README.md

```
@@ -70,3 +70,86 @@ mermaid.registerLayoutLoaders(elkLayouts);
+elk: {
+  algorithm: 'elk.force' | 'force'; // Algorithm identifier
+  forceModel: 'FRUCHTERMAN_REINGOLD' | 'EADES'; // Force model variant (default: FRUCHTERMAN_REINGOLD)
+  forceIterations: number; // Number of layout iterations (default: 300, minimum: 1)
+  forceRepulsion: number; // Repulsive force between nodes (default: 5.0)
+  forceTemperature: number; // Initial temperature for node movement (default: 0.001)
+}
+```
+
+**Example:**
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)